pages/posts/w3c-wot-case-study.md

```diff
@@ -0,0 +1,152 @@
+explore them further in conjunction with Semantic Web technologies."
+
+## Impact
+
+The W3C Web of Things relies on JSON-LD for linking vocabularies and ontologies
+that already exist on the web, such as [SAREF][saref], [Units of
+Measure][units-of-measure] and [Schema.org][schema-org]. Being based on the
+JSON data model, JSON Schema fits well with JSON-LD when integrating both
+technologies as part of a single specification.
+
+Adopting JSON Schema means that the W3C Web of Things does not need to invest
+effort on inventing another schema language. As a consequence of the popularity
+of JSON Schema, existing tooling can be often reused instead of implementing
+custom parsers, validators, code generators and UI generators for all the
+programming languages that the community might be interested in.
+
+JSON Schema has proven to be a foundational block for creating higher-level
+tooling that is specific to the W3C Web of Things. As a notable example, the
+community has developed a tool called
+[testbench](https://github.com/tum-esi/testbench) that makes use of JSON Schema
+definitions inside Thing Descriptions to generate matching payloads for the
+purpose of stress and penetration testing.
+
+Given the growing popularity of using JSON Schema to generate web-based forms,
+W3C Web of Things specifications can be used to automatically generate
+dashboards and user interfaces to interact with the objects modeled by Thing
+Descriptions.
+
+## Key Impact Results
+
+The ultimate metric of success for a standard like the W3C Web of Things is
+adoption, and community adoption is not something you can just "engineer".  Ege
+Korkan commented, "since we adopt JSON Schema, developers who are familiar
+with it have a smaller learning curve when learning the W3C Web of Things
+standard."
+
+"During my PhD times, my MSc engineering students had to become familiar with
+the W3C Web of Things. These students typically took 1 week to go through my
+[tutorial](https://github.com/tum-esi/wot-tutorial). If they knew JSON Schema,
+that 1 week would become 2 days", said Ege Korkan.
+
+## W3C Web of Things
+
+The World Wide Web Consortium (W3C) is the standardization organization behind
+most of the standards used in the Web. We are the Web of Things Working and
+Interest Groups, who work on the standardization of Web of Things at W3C.
+
+The Web of Things groups aim to create an interoperability layer so that
+different Internet of Things platforms, protocols and standards can operate
+together.
+
+The work has started as a discussion in the Web of Things Community Group in
+2013-2014.  It has proceeded to become an Interest Group in 2015 that has
+collected the use cases and defined the standards to be worked on. Since 2016,
+the working group is working on different standards on the Web of Things with
+the first publications of the Thing Description and Architecture
+recommendations in 2019 and more that will be published by the end of this
+year.
```

Comment:
```suggestion
The work started as a discussion in the Web of Things Community Group in
2013-2014.  It proceeded to become an Interest Group in 2015 that
collected the use cases and defined the standards to be worked on. Since 2016,
the working group is working on different standards on the Web of Things with
the first publications of the Thing Description and Architecture
recommendations in 2019 and more that will be published by the end of this
year.
```

Minor gramatical corrections.